_posts/news/2016-10-17-announcing-scalajs-0.6.13.md

```diff
@@ -0,0 +1,164 @@
+
+* Node.js by default
+* Node.js + [`jsdom`](https://github.com/tmpvar/jsdom) with `jsDependencies += RuntimeDOM`
+
+Note that Node.js and jsdom need to be installed separately on your system.
+
+* [Download Node.js here](https://nodejs.org/en/download/)
+* Install jsdom with `npm install jsdom` (either in your project's root directory, or [globally](https://docs.npmjs.com/getting-started/installing-npm-packages-globally))
+
+We decided to standardize on Node.js for all command-line executions by default because it is always the best alternative.
+Rhino is extremely slow.
+Using by default, although working out-of-the-box, caused a number of users to stick to it and discovering months or years later that their tests could run 10x-100x faster by switching to Node.js.
+PhantomJS, on the other hand, [reportedly suffers from significant issues](https://github.com/scala-js/scala-js/issues/1881).
+`jsdom` seems to be better maintained at this point.
+
+You can restore the previous behaviors with the following sbt settings:
+
+* Rhino: `scalaJSUseRhino in Global := true`.
+  Rhino is however *deprecated*, and will not be supported anymore in 1.0.0.
+* PhantomJS: `jsEnv := PhantomJSEnv().value`.
+
+Also, remember that you can also use Selenium with Scala.js, using [scalajs-env-selenium](https://github.com/scala-js/scala-js-env-selenium).
+
+## Deprecation: `@JSGlobalScope` replaces `extends js.GlobalScope`
+
+As indicated by a deprecation warning, `extends js.GlobalScope` should not be used anymore.
+Instead, you should annotate the object with `@JSGlobalScope`.
+For example, this old snippet:
+
+{% highlight scala %}
+import scala.scalajs.js
+
+@js.native
+object Foo extends js.GlobalScope
+{% endhighlight %}
+
+should be replaced with
+
+{% highlight scala %}
+import scala.scalajs.js
+import js.annotation._
+
+@js.native
+@JSGlobalScope
+object Foo extends js.Object
+{% endhighlight %}
+
+## `@JSImport` and emitting CommonJS modules
+
+This is a long-awaited feature!
+Scala.js can now emit CommonJS modules (i.e., those used by Node.js, as well as several bundlers).
+
+### Enabling CommonJS module
+
+You can enable emission of a CommonJS module with the following sbt setting:
+
+{% highlight scala %}
+scalaJSModuleKind := ModuleKind.CommonJSModule
+{% endhighlight %}
+
+When emitting a CommonJS module, top-level `@JSExport`ed classes and objects are stored in the `exports` object, so that they can be required from other CommonJS modules.
+
+Obviously, this requires that you use Node.js for `sbt run` and `sbt test` (should you use them at all), and that you use a JavaScript bundler such as [Webpack](https://webpack.github.io/docs/) to create a bundle fit for use in the browser.
+At the moment, Scala.js does not provide any facility to do so.
+
+Emitting CommonJS modules is also not compatible with `persistLauncher := true`, as a different launcher needs to be emitted for fastOpt versus fullOpt.
+
+### Importing stuff from other CommonJS modules
```

Comment:
You absolutely should add a disclaimer here, that `jsDependencies` support for this is pending and that this will only work for builds that manage their JS dependencies externally.